### DISCUSSION TITLE
Gui

### DISCUSSION THREAD

**Discussion by Nichtraucher** [marcosnils/bin#182] [Ideas]
Hey,

I'd appreciate a Gui very much. Perhaps something similar to [Obtainium](https://github.com/ImranR98/Obtainium) for Android?

cheers

**Comment by marcosnils** [comment]
hey! thx for the feedback! Unfortunately, I personally don't have the time or the need to build a GUI. If you or someone feels like giving a try, happy to provide some support and feedback during the process.
